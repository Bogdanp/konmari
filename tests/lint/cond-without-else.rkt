#lang racket/base

(define x 0)

(cond
  [(> x 0) 'positive])

(define res
  (cond
    [(> x 0) 'positive]))

(define (f x)
  (cond
    [(> x 0) 'positive]))

(cond
  [(void)])

(cond
  [#f 1]
  [else 2])
