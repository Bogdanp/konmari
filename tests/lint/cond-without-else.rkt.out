/cond-without-else.rkt:5:1:warning:this cond expression does not have an else clause
/cond-without-else.rkt:9:3:warning:this cond expression does not have an else clause
/cond-without-else.rkt:12:12:warning:identifier 'x' shadows an earlier binding
/cond-without-else.rkt:13:3:warning:this cond expression does not have an else clause
/cond-without-else.rkt:8:9:warning:identifier 'res' is never used
/cond-without-else.rkt:12:10:warning:identifier 'f' is never used
